RecoLocalTracker/SiPixelRecHits/src/SiPixelTemplateReco2D.cc

```diff
@@ -0,0 +1,654 @@
+               clusxy[j-1][i] = pseudopix;
+               pixel[tpixel] = pseudopix;
+               sigma2[tpixel] = pseudopix2;
+               ++tpixel;
+            }
+         }
+         if((j+1) != jmisslow) {
+            if(clusxy[j+1][i] < pseudopix) {
+               indexxy[0][tpixel] = j+1;
+               indexxy[1][tpixel] = i;
+               clusxy[j+1][i] = pseudopix;
+               pixel[tpixel] = pseudopix;
+               sigma2[tpixel] = pseudopix2;
+               ++tpixel;
+            }
+         }
+         // Don't add them if this is a dead column
+         if((i+1) != imisslow) {
+            if((j-1) != jmisshigh) {
+               if(clusxy[j-1][i+1] < pseudopix) {
+                  indexxy[0][tpixel] = j-1;
+                  indexxy[1][tpixel] = i+1;
+                  clusxy[j-1][i+1] = pseudopix;
+                  pixel[tpixel] = pseudopix;
+                  sigma2[tpixel] = pseudopix2;
+                  ++tpixel;
+               }
+            }
+            if(clusxy[j][i+1] < pseudopix) {
+               indexxy[0][tpixel] = j;
+               indexxy[1][tpixel] = i+1;
+               clusxy[j][i+1] = pseudopix;
+               pixel[tpixel] = pseudopix;
+               sigma2[tpixel] = pseudopix2;
+               ++tpixel;
+            }
+            if((j+1) != jmisslow) {
+               if(clusxy[j+1][i+1] < pseudopix) {
+                  indexxy[0][tpixel] = j+1;
+                  indexxy[1][tpixel] = i+1;
+                  clusxy[j+1][i+1] = pseudopix;
+                  pixel[tpixel] = pseudopix;
+                  sigma2[tpixel] = pseudopix2;
+                  ++tpixel;
+               }
+            }
+         }
+         // Don't add them if this is a dead column
+         if((i-1) != imisshigh) {
+            if((j-1) != jmisshigh) {
+               if(clusxy[j-1][i-1] < pseudopix) {
+                  indexxy[0][tpixel] = j-1;
+                  indexxy[1][tpixel] = i-1;
+                  clusxy[j-1][i-1] = pseudopix;
+                  pixel[tpixel] = pseudopix;
+                  sigma2[tpixel] = pseudopix2;
+                  ++tpixel;
+               }
+            }
+            if(clusxy[j][i-1] < pseudopix) {
+               indexxy[0][tpixel] = j;
+               indexxy[1][tpixel] = i-1;
+               clusxy[j][i-1] = pseudopix;
+               pixel[tpixel] = pseudopix;
+               sigma2[tpixel] = pseudopix2;
+               ++tpixel;
+            }
+            if((j+1) != jmisslow) {
+               if(clusxy[j+1][i-1] < pseudopix) {
+                  indexxy[0][tpixel] = j+1;
+                  indexxy[1][tpixel] = i-1;
+                  clusxy[j+1][i-1] = pseudopix;
+                  pixel[tpixel] = pseudopix;
+                  sigma2[tpixel] = pseudopix2;
+                  ++tpixel;
+               }
+            }
+         }
+      }
+      
+      
+      //   		 printf("** 2D-cluster, cotalpha = %f, cotbeta = %f **\n", cotalpha, cotbeta);

```

Comment:
Are these commented out lines needed? If so, please add another line of comment which explain why; otherwise just remove them

Reply:
Is this about the printf statement?

Reply:
All lines between 421 and 428 are now commented out and not used